system/conf/login.conf

```diff
@@ -2,6 +2,9 @@
 // Configuration file
 // --------------------------------------------------------------------------
 
+// IPF MD5 checksum
+ipf_checksum: ce9d9325d0fd35ceae14ce5261d75c98
```

Comment:
This needs to be generated any time specific IPF archives change. In my own extractor, I generate this one time. I'm trying to think what would be appropriate for Melia...

Probably the easiest thing to do would be to add the generation code to IPFBrowser.

Reply:
I usually don't include checks like these if it means that you have to update it regularly, especially if it's not easy to obtain for the common user. Do you think that's worth the trouble? I think it should probably at least be optional, like don't check if the option is empty or something. What do you think?

Reply:
If it is optional, I think that is fine. I think it is more important that the functionality is documented and exists, so that someone isn't left wondering what this part of the packet represents or is used for.